### PR TITLE
renames in Queue API

### LINKED ISSUE #8151: [P0] Storage: Rename QueueClient methods and response types
https://apiview.azurewebsites.net/Assemblies/Review/231f6f1802d44c8e9f7c1bfe0819a413

Rename Enqueue/Dequeue methods and all response types to:
- `Response<SendMessageResult> SendMessage(…)`
- `Response<QueueMessageItem[]> ReceiveMessages(…)`
- `Response<PeekedMessageItem[]> PeekMessages(…)`
- `Response<UpdateMessageResult> UpdateMessage(…)`
- `Response DeleteMessage(…)`

### PR DESCRIPTION
**Pull request by JoshLove-msft**
Fixes #8151 and #8153

### CODE REVIEW

File: sdk/storage/Azure.Storage.Queues/swagger/readme.md [hunk 1]
```diff
@@ -226,36 +226,36 @@ directive:
...
     const def = $.items;
-    if (!def["$ref"].endsWith("DequeuedMessage")) {
-        const path = def["$ref"].replace(/[#].*$/, "#/definitions/DequeuedMessage");
+    if (!def["$ref"].endsWith("QueueMessageItem")) {
+        const path = def["$ref"].replace(/[#].*$/, "#/definitions/QueueMessageItem");
         $.items = { "$ref": path };
     }
 ```
 
-### PeekedMessage
```
Comment (tg-msft) [inline sdk/storage/Azure.Storage.Queues/swagger/readme.md]: We're getting rid of all the Peeked transformations?

**Review by tg-msft — APPROVED** [review]
This looks good

**Review by amnguye — APPROVED** [review]

**Review by tg-msft — APPROVED** [review]

**Review by tg-msft — APPROVED** [review]

**Comment by seanmcc-msft** [comment]
Couldn't this introduce some ambiguity for the user?  From who's prospective are we sending the receiving the message, the user's prospective, or the queue's prospective?

**Comment by tg-msft** [comment]
I don't think so - we haven't seen that problem with all the other messaging APIs that we're modeling this after (i.e., ServiceBus).

**Comment by JoshLove-msft** [comment]
Is this good to merge? @seanmcc-msft @tg-msft

**Comment by tg-msft** [comment]
Let's wait a bit to make sure @seanmcc-msft doesn't have any more feedback, but otherwise we should merge it this morning.

**Review by amnguye — APPROVED** [review]

**Comment by seanmcc-msft** [comment]
Let's go ahead and merge.  If this naming schema works for Service Bus, it works for us.

**Pull request merged**